lib/twilio-ruby/rest/client.rb

```diff
@@ -1,3 +1,5 @@
+require 'logger'
```

Comment:
Is this needed? I think it can be removed and the dependency moved to a dev-only dependency.